tabify/fecHeatmapTabify.js

```diff
@@ -29,27 +31,27 @@ export default class FecHeatmapTabify {
             key >= 1.5 && key < 2.0 ? '1.5% - 1.99%' :
             key >= 2.0 && key < 4.0 ? '2.0% - 3.99%' : 
             key >= 4.0 && key < 6.0 ? '4.0% - 5.99%' : 
-            key >= 6.0 && key < 10.0 ? '6.0% - 9.99%' : '>= 10.0%';
+            key >= 6.0 && key < 10.0 ? '6.0% - 9.99%' : 
+            key >= 10.0 ? '>= 10.0%' : NO_DATA;
     }
     
     process(response) {
         const aggregations = response && response.aggregations;
-        const NO_DATA = 'No Data';
         if (!isEmpty(aggregations)) {
             const result = [];
             if (aggregations.date_histo && aggregations.date_histo.buckets) {
                 for (const dateHistoEntry of aggregations.date_histo.buckets) {
-                    const networkLossValue = dateHistoEntry.NetworkLoss ? dateHistoEntry.NetworkLoss.value : NO_DATA;
-                    const lossAfterFecValue = dateHistoEntry.LossAfterFEC ? dateHistoEntry.LossAfterFEC.value : NO_DATA;
+                    const networkLossValue = dateHistoEntry.NetworkLoss && dateHistoEntry.NetworkLoss.value ? dateHistoEntry.NetworkLoss.value : NO_DATA;
```

Comment:
This will not consider zero value. If dateHistoEntry.NetworkLoss.value is 0, this will result in No Data.
Pls add a new util function in js-bambou, on the lines of `isVariableNonNullAndDefined` in vsd-react-ui and use the same

Reply:
Done.